com.github.scrivanolabs.scrivano.yml

```diff
@@ -0,0 +1,40 @@
+app-id: com.github.scrivanolabs.scrivano
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: Scrivano
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --device=dri
+modules:
+  - name: Scrivano
+    buildsystem: simple
+    build-commands:
+    - install -D Scrivano /app/bin/Scrivano
+    - install -D libskia.so /app/lib/libskia.so
+    - install -D libpdfium.so /app/lib/libpdfium.so
+    - install -Dm644 com.github.scrivanolabs.scrivano.appdata.xml -t /app/share/metainfo
+    - install -Dm644 com.github.scrivanolabs.scrivano.desktop -t /app/share/applications
+    - install -Dm644 scrivano_64.png /app/share/icons/hicolor/64x64/apps/com.github.scrivanolabs.scrivano.png
+    - install -Dm644 scrivano_128.png /app/share/icons/hicolor/128x128/apps/com.github.scrivanolabs.scrivano.png
+    - install -Dm644 scrivano_256.png /app/share/icons/hicolor/256x256/apps/com.github.scrivanolabs.scrivano.png
+    - install -Dm644 scrivano_512.png /app/share/icons/hicolor/512x512/apps/com.github.scrivanolabs.scrivano.png
+    sources:
+    - type: archive
+      url: https://github.com/scrivanolabs/ScrivanoForLinux/releases/download/0.13.4/Scrivano_flatpak_files_0.13.4.zip
+      sha256: 46a79db8cb96eb9ad9a089d487bc7d97d1c585b17fb08b35ecd8e3cc9460708e
+    - type: file
+      path: icons/scrivano_64.png
```

Comment:
why aren't the icons in the downloaded archive?

Reply:
also why is the included svg never installed?

Reply:
The build fail because the appdata file has:
```
  <launchable type="desktop-id">com.github.scrivano.scrivano.desktop</launchable>
```
which is incorrect.

Reply:
For some reason, when I tested locally I was getting a strange `error: syncfs: Input/output error` which disappeared when I moved the icons in the repo. Anyway, I tested again and moved the icons to the installation source and it seems to work correctly.

Reply:
> also why is the included svg never installed?

Fixed.

Reply:
> The build fail because the appdata file has:
> 
> ```
>   <launchable type="desktop-id">com.github.scrivano.scrivano.desktop</launchable>
> ```
> 
> which is incorrect.

Thanks for pointing this out. I fixed this.